src/queryNames.js

```diff
@@ -0,0 +1,27 @@
+function constructQueries() {
+  const strategies = [
+    "LabelText",
+    "PlaceholderText",
+    "Text",
+    "AltText",
+    "Title",
+    "DisplayValue",
+    "Role",
+    "TestId",
+  ];
+
+  const by = [
+    "getBy",
+    "getAllBy",
+    "queryBy",
+    "queryAllBy",
+    "findBy",
+    "findAllBy",
+  ];
+
+  return by.reduce((acc, q) => {
```

Comment:
rather than hardcoding these here, can we just import queries from dom-testing-library instead?  in fact if we do that i think this whole file can be removed and replaced with `import { queries } from "@testing-library/dom";`

Reply:
Ah, that's way better. I had to disable the lint rule `testing-library/no-import/dom` to pass this. That's fine, or no?

Reply:
totally fine. 👍